src/app/participant/survey-registration/survey-registration.component.ts

```diff
@@ -66,13 +66,13 @@ export class SurveyRegistrationComponent {
   private fetchSurvey() {
     this.surveyService.getSurvey(this.surveyCode).subscribe((surveyDto) => {
       this.survey = surveyDto
-      this.fetchEvent(this.survey.eventId, true);
+      this.shouldNavigate(this.survey.eventId, true);
     }, (error) => {
       this.router.navigate(['/404'])
     })
   }
 
-  private fetchEvent(eventId: number, navigate: boolean): void {
+  private shouldNavigate(eventId: number, navigate: boolean): void {
```

Comment:
A small misunderstanding here, I meant that the parameter "navigate" should be named "shouldNavigate".
Sorry for confusion! 😀

Reply:
Oh my god